contracts/ResultManager.sol

```diff
@@ -72,42 +67,35 @@ contract ResultManager is AccessControlEnumerable {
         );
 
         uint16[] memory ids = new uint16[](values.length);
```

Comment:
this can be removed

Reply:
Resolved in [780bf9d](https://github.com/razor-network/bridge-proxy/pull/50/commits/780bf9de15de5fe8f5d208cddcf65a72dd79a60d)